aten/src/ATen/native/quantized/cudnn/Conv.cpp

```diff
@@ -377,7 +392,6 @@ class QConvInt8 final {
       const c10::intrusive_ptr<ConvPackedParamsBase<kSpatialDim>>& packed_weight,
       double output_scale,
       int64_t output_zero_point) {
-    act = act.to(c10::MemoryFormat::ChannelsLast);
```

Comment:
would be worthwhile to document the expected memoryformat for packed_weight->apply/apply_relu for input I think, also document the memoryformat for the packed weight as well (in packedweight class)